### DISCUSSION TITLE
Dogecoin Core 1.14.9

### DISCUSSION THREAD

**Discussion by patricklodder** [dogecoin/dogecoin#3731] [General]
This is a new minor version release that fixes bugs that were inherited from upstream Bitcoin and Namecoin code.

***All users, on all platforms, are strongly recommended to upgrade.***

Please see the [release notes](https://github.com/dogecoin/dogecoin/blob/e0a1c157791544e818c901bd9341896965afbf9d/doc/release-notes.md) for further information.

<hr /><em>This discussion was created from the release <a href='https://github.com/dogecoin/dogecoin/releases/tag/v1.14.9'>Dogecoin Core 1.14.9</a>.</em>

**Comment by trpbillionairedubai** [comment]
Excellent opportunity to beat Bitcoin through Doge coin

**Reply by Jloshow** [reply]
Can someone please explain in detail how this works for me.

**Reply by okutolllwn3llll** [reply]
HONOR

**Reply by Jloshow** [reply]
This is not actual coding is it get hub is a tribute to the existing code

**Reply by imanilie** [reply]
> Excellent opportunity to beat Bitcoin through Doge coin

**Reply by kalista1** [reply]
How to download this app?

**Reply by Hasnanst** [reply]
Halo 👋👍

**Reply by uribepachecoluisantonio-lang** [reply]
Gracias 🫰😂

**Comment by Deluciferrral** [comment]
- [ ] H[]()

**Reply by oning** [reply]
Why can't I withdraw from my dogecoin Wallet?

**Reply by oning** [reply]
I don't understand how to download dogecoin Core wallet too complicated in this Reddit

**Comment by shajiv1** [comment]
Ho

**Comment by shajiv1** [comment]
Hi

**Reply by Deluciferrral** [reply]
#^&^# 
/<>hello)

**Comment by Deluciferrral** [comment]
ok so how this works...

**Comment by rimonhassan0** [comment]
Hi

**Reply by Rahmanmhmd** [reply]
- [ ] []()

**Comment by RuskoxXx3488** [comment]
To the moooooon

**Comment by Nino4441** [comment]
❤️

პარ, 4 აპრ. 2025, 11:07-ში ابو كيان ***@***.***> დაწერა:

**Comment by hungtoro** [comment]
sha256:8dcedfb86ee467f965c03fabb7819eba21495bee43db6bc0ae5c7b2f94c2b54a

**Reply by okutolllwn3llll** [reply]
[]()```

```

**Comment by Rahmanmhmd** [comment]
Rahmanbusiness

**Comment by Sahid9120** [comment]
Hii Dogecoin..

**Reply by Jloshow** [reply]
How are you

**Reply by Rahmanmhmd** [reply]
Ok

**Reply by Deluciferrral** [reply]
Good

**Reply by Deluciferrral** [reply]
Good

**Comment by Deluciferrral** [comment]
im good wbu

**Comment by Jloshow** [comment]
Looking for funding and a tutor

**Reply by marssystems** [reply]
> Looking for funding and a tutor On Tue, Apr 29, 2025, 4:05 AM Deluciferrral ***@***.***> wrote:
> […](#)
> im good wbu On Tue, Apr 22, 2025 at 9:05 AM Jloshow ***@***.***> wrote: > How are you > > — > Reply to this email directly, view it on GitHub > < [#3731 (reply in thread)](https://github.com/dogecoin/dogecoin/discussions/3731#discussioncomment-12911078)>, > or unsubscribe > < https://github.com/notifications/unsubscribe-auth/AWYWB6EGUWUEPXXTCXM6HZL22ZEBBAVCNFSM6AAAAABSZ2GZ6GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJRGEYDOOA> > . > You are receiving this because you commented.Message ID: > ***@***.***> > — Reply to this email directly, view it on GitHub <[#3731 (comment)](https://github.com/dogecoin/dogecoin/discussions/3731#discussioncomment-12978505)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BAJ6WEH4XLQ7SAHHAYGHKPT2346GFAVCNFSM6AAAAABSZ2GZ6GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJXHA2TANI> . You are receiving this because you commented.Message ID: ***@***.***>

Funding for what ??

**Comment by Deluciferrral** [comment]
if your asking me then yes im looking'

**Reply by Jloshow** [reply]
Same

**Comment by gifted110** [comment]
Hello 👋

**Reply by kashem6s** [reply]
Hi

**Comment by gifted110** [comment]
Good morning sir 👍

**Comment by MD10inoffun** [comment]
Hello world

**Comment by gifted110** [comment]
Hello 👋 sir

**Comment by Deluciferrral** [comment]
Hello from USA Sent from my iPhoneOn Oct 24, 2025, at 6:49 AM, DaneLand-ctrl ***@***.***> wrote:﻿
Hallo from Denmark 🇩🇰 😀

**Comment by DaneLand-ctrl** [comment]
Hallo from Denmark, posting positive thoughts to everyone ✨️ 😊 😀

**Comment by FrosTether** [comment]
God bless you dogie

**Reply by Jennie867-5309** [reply]
￼❤️

**Comment by keepsmile8** [comment]
where is a version for mac os

**Reply by John-Inubook** [reply]
https://github.com/dogecoin/dogecoin/releases/

**Comment by uribepachecoluisantonio-lang** [comment]
🤙[]()[]()![IMG-20260209-WA0028.jpg](https://github.com/user-attachments/assets/641f4097-4882-4100-8725-3f217c9844d7)

**Reply by Jennie867-5309** [reply]
So Beautiful. ❤️

**Comment by Jennie867-5309** [comment]
Thanks! ❤️


Sent from Yahoo Mail for iPhone


On Wednesday, April 1, 2026, 8:12 AM, uribepachecoluisantonio-lang ***@***.***> wrote:




🤙IMG-20260209-WA0028.jpg (view on web)

**Comment by ihuicn** [comment]
很多国外公司和清华和外包hehhsh

**Reply by uribepachecoluisantonio-lang** [reply]
Hola
